README.md

```diff
@@ -66,7 +66,7 @@ $ puppet-query 'nodes[certname]{}'
     "certname" : "foo.example.com"
   }
 ]
-$ puppet-db status
+$ puppet-db status # Note `puppet-db status` will be released in puppet-client-tools 1.1.0
```

Comment:
Yeah a ticket came in where someone was confused about the status of status :) so I thought I'd add a note just to clarify.